stories/features/rewards/other.tsx

```diff
@@ -181,7 +181,7 @@ storiesOf('Feature Components/Rewards/Other/Desktop', module)
           type={select<any>('Type', { big: 'big', small: 'small' }, 'big')}
           title={'Jonathon Doe'}
           verified={boolean('Verified', false)}
-          provider={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch' }, 'youtube')}
+          provider={select<any>('Provider', { youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch', reddit: 'Reddit' }, 'youtube')}
```

Comment:
Can you please change `{ youtube: 'YouTube', twitter: 'Twitter', twitch: 'Twitch', reddit: 'Reddit' }` to `{ youtube: 'youtube', twitter: 'twitter', twitch: 'twitch', reddit: 'reddit' }`

Reply:
Done